main.go

```diff
@@ -123,13 +124,25 @@ func main() {
 			setupLog.Error(err, "unable to start cluster aware manager")
 			os.Exit(1)
 		}
+
+		// Local client is needed to access the workspace where operator deployment is
+		localClientSceme := runtime.NewScheme()
```

Comment:
```suggestion
		localClientScheme := runtime.NewScheme()
```

typo

Reply:
@maysunfaisal thanks. Fixed. Could you merge it? (I have no permissions)